docs/tutorial/04_integrate_sdcore_with_observability.md

```diff
@@ -0,0 +1,81 @@
+# 5. Integrate SD-Core with Observability
+
+We will integrate the 5G core network with the Canonical Observability Stack (COS).
+
+## Enable the `metallb` MicroK8s addon
+
+Enable the `metallb` addon with a range of 1 IP address:
+
+```console
+sudo microk8s enable metallb 10.0.0.2-10.0.0.2
```

Comment:
Can this be just any ip? if not, should we add more instructions?

Reply:
Yes it can be any IP, here I'm forcing it to a specific IP because I'll re-use it to tell the user to connect to this address on their browser.

Reply:
Got it :)